questionpy_sdk/webserver/routes/attempt.py

```diff
@@ -94,7 +94,7 @@ async def get_attempt(request: web.Request) -> web.Response:
     return aiohttp_jinja2.render_template("attempt.html.jinja2", request, context)
 
 
-async def _score_attempt(request: web.Request, data: dict) -> web.Response:
+async def _score_attempt(request: web.Request, data: Any) -> web.Response:
```

Comment:
Hier würde ich lieber den Typen prüfen (Entweder mit `isinstance` oder Pydantic) als zu entfernen. Kann ich mich aber auch gern drum kümmern, wenn du den PR schnell durch haben möchtest.

Reply:
> Typen prüfen (Entweder mit isinstance oder Pydantic)

Hmm... ich weiß leider nicht genau, was damit gemeint ist, aber der Commit fixt nicht die Probleme in `attempt.py`, sondern lediglich die Probleme, die in #123 benannt sind, so dass die Projekte hoffentlich bald wieder konsistent und lauffähig sind.

M.E.n. ist der Type Error, der in der Python 3.11/mypy 1.12.0-Kombi auftritt korrekt. Der Rückgabewert von `Request.json` ist halt `Any`. Der präferierte Fix wäre eine Schemavalidierung der Daten zur Laufzeit, z.B. mittels des Pydantic-JSON-Parsers. Aber das ist nicht der Scope dieses Issues...

Ich hoffe, ich habe dein Anliegen richtig verstanden.